src/main/java/io/neonbee/endpoint/raw/RawEndpoint.java

```diff
@@ -88,6 +88,8 @@ public Future<Router> createEndpointRouter(Vertx vertx, String basePath, JsonObj
     static class RawHandler implements Handler<RoutingContext> {
         private static final String CONTENT_TYPE = "Content-Type";
 
+        private static final String RESPONSE_HEADERS = "RESPONSE_HEADERS";
```

Comment:
I think it makes sense to make this public, because consumers need to set the value in their verticles and they should use the constant instead of had coding the value. Also I would move it to Endpoint.java, as it is relevant for all endpoints. Also I would call it "RESPONSE_HEADERS_HINT", as we agreed that everything that the endpoints evaluate we treat as a "hint" towards the endpoints.